code/src/App.js

```diff
@@ -1,9 +1,120 @@
-import React from 'react';
+/* eslint-disable react/jsx-props-no-spreading */
+import React, { useState } from 'react';
+import Startpage from 'components/Startpage';
+import Types from 'components/Types';
+import Summary from 'components/Summary';
+import './index.css';
+
+/* You can add more questions in this array, or move the content to a file and import it */
+
+const questionsData = [
+  {
+    id: '0'
+  },
+  {
+    id: '1',
+    questionTitle: "Nice, let's do this!",
+    questionText: "First of all, what's your name?",
+    inputType: 'text'
+  },
+  {
+    id: '2',
+    questionTitle: 'ENERGY LEVEL',
+    questionText: 'Energywise, how much energy do you need?',
+    inputType: 'select',
+    options: [
+      { value: 'A lot' },
+      { value: 'Just enough to keep me going' },
+      { value: 'Low key, something slow' }
+    ]
+  },
+  {
+    id: '3',
+    questionTitle: 'GENRE',
+    questionText: 'What type of music do you prefer?',
+    inputType: 'radio',
+    options: [
+      { value: 'Pop/Rock', label: 'Pop/Rock' },
+      { value: 'Hiphop', label: 'Hiphop' },
+      { value: 'World', label: 'World' }
+    ]
+  },
+  {
+    id: '4',
+    questionTitle: 'EPOKE',
+    questionText: 'Do you prefer contemporary music or are oldies?',
+    inputType: 'radio',
+    options: [
+      { value: 'Contemporary', label: 'Contemporary' },
+      { value: 'Oldies', label: 'Oldies' }
+    ]
+  }
+];
 
 export const App = () => {
+  const [currentQuestion, setcurrentQuestion] = useState(0);
+  const [userInput, setUserInput] = useState('');
+  const [answers, setAnswers] = useState([]);
+
+  /// //Keep track of which question is next////
+  const handleFormSubmit = (e) => {
+    e.preventDefault();
+    const currentQuestionNumber = currentQuestion + 1;
+    setcurrentQuestion(currentQuestionNumber);
+
+    /// //Save answers////
+    const updateAnswer = (updatedAnswer) => {
+      setAnswers(() => [...answers, ...updatedAnswer]);
```

Comment:
Haven't seen "..." om row 67 before, curious of what it means :)

Reply:
Yes, it's something a friend of mine showed me. :) It's just a short-cut. Here is example from stack overflow:



<img width="682" alt="Screenshot 2022-10-05 at 15 23 29" src="https://user-images.githubusercontent.com/108176641/194071095-92fa2402-5c57-4978-841f-7b638f133821.png">